icu4j/main/tests/core/src/com/ibm/icu/dev/test/util/DebugUtilitiesData.java

```diff
@@ -7,7 +7,7 @@
 package com.ibm.icu.dev.test.util;
 
 public class DebugUtilitiesData extends Object {
-    public static final String ICU4C_VERSION="50.1.1";
+    public static final String ICU4C_VERSION="50.1.2";
```

Comment:
50.1.2?

Reply:
I think this is the result of ICU4J data re-generation. I think ICU4J did not really have 50.1.2, but C did.

Anyway, I'll add one more commit for version number, and sync data between C and J, that will update this file again.

Reply:
It's in the maint-50 branch, the ICU4C version was actually bumped to 50.1.2 on the branch/.